server/data/database.js

```diff
@@ -0,0 +1,80 @@
+/* @flow */
+
+// mock data
+export class Todo {
+  id: string;
+  content: string;
+}
+export class User {
+  id: string;
+}
+
+// main viewer (default user)
```

Comment:
can you expand the description a little on what the main viewer/default user is and why it exists for this app?